workspace/TS100/Core/Src/TipThermoModel.cpp

```diff
@@ -73,17 +73,19 @@ int32_t LinearInterpolate(int32_t x1, int32_t y1, int32_t x2, int32_t y2,
 uint32_t TipThermoModel::convertuVToDegC(uint32_t tipuVDelta) {
 	//based on new measurements, tip is quite linear
 	//
-	tipuVDelta *= TIP_GAIN;
-	tipuVDelta /= 10000;
+	tipuVDelta *= 100;
+	tipuVDelta /= systemSettings.TipThermoCoeff;
+
+#ifdef TS80
+	tipuVDelta *= 100 / 349; // TS100 op_amp_gain_stage / TS80 op_amp_gain_stage
```

Comment:
`349` is a magic number and should come from an `#ifdef` somewhere if you can

Reply:
Secondly, use the model as like a switch case. Dont use modifier magic numbers (please)

```
#ifdef MODEL_TS100
#else
#endif
```